quality_control_oca/views/qc_trigger_view.xml

```diff
@@ -26,7 +26,7 @@
         <field name="name">qc.trigger.tree</field>
         <field name="model">qc.trigger</field>
         <field name="arch" type="xml">
-            <tree editable="bottom">
+            <tree>
```

Comment:
Why removing the editable thing?

Reply:
There is a view form that is not used, and I add picking type view to this field, to check if picking type is right... this will be also usefull if you have more that one warehouse with same picking type names.

Reply:
ok, finally I update only quality_control_stock_oca module, after this PR I can do other with other improvements